### ISSUE TITLE
Bug: ClearFilterCommand does not clear FilterTemplates

### ISSUE THREAD

**Issue by DDDenisSobek** [stsrki/Blazorise#1150]
**Describe the bug**
If `FilterTemplate` is defined and ClearFilterCommand is fired, filters will be removed, but old inputs bill stay in filter inputs.

**To Reproduce**
1. Go to [DataGridPage](https://bulmademo.blazorise.com/tests/datagrid)
2. Filter with "First Name" and "City" ([this columns has `FilterTemplates`](https://github.com/stsrki/Blazorise/blob/master/Demos/Blazorise.Demo/Pages/Tests/DataGridPage.razor))
3. Click on 'Clear Filter'
4. See that filtering is gone, but your input stays

**Expected behavior**
Filter Templates will be reinitialize, so that old inputs removed by clear filter.

**Screenshots**
![grafik](https://user-images.githubusercontent.com/65016836/87656200-76f4e880-c759-11ea-88b6-e788de28fd41.png)
![grafik](https://user-images.githubusercontent.com/65016836/87656259-8b38e580-c759-11ea-8f3c-c4b485f7b008.png)
![grafik](https://user-images.githubusercontent.com/65016836/87656298-968c1100-c759-11ea-8ce4-d5776c06ebf3.png)

**Comment by stsrki** [comment]
How do you bind values of input field and the filter context?

**Comment by DDDenisSobek** [comment]
More or less like ['First Name' in DataGridPage](https://github.com/stsrki/Blazorise/blob/master/Demos/Blazorise.Demo/Pages/Tests/DataGridPage.razor).
```
...
<FilterTemplate>
    <NumericEdit TValue="int?" Step="1" Decimals="0" Value="@Get(context)"
                            ValueChanged="@(value => Changed(value, context))" />
</FilterTemplate>
...
protected int? Get(FilterContext context)
{
       var searchValue = context.SearchValue;
       var hasFilter = int.TryParse(searchValue, out var filter);
       int? value = null;
       if (hasFilter)
       {
           value = filter;
       }
       return value;
}
protected void Changed(int? value, FilterContext context)
{
    var searchValue = value?.ToString();
    context.SearchValue = searchValue;
    context.TriggerFilterChange(searchValue);
}
```

**Comment by DDDenisSobek** [comment]
@stsrki Do i something wrong?

**Comment by stsrki** [comment]
@DDDenisSobek I reproduced it on my side and it seems it's a bug. I will put on backlog for next milestone.

**Comment by stsrki** [comment]
Already fixed with #1719
